### ISSUE TITLE
Feature Suggestion: Add ZVAL to POTCAR metadata

### ISSUE THREAD

**Issue by arosen93** [materialsproject/pymatgen#2540]
Currently, the `Potcar()` object in pymatgen does not store the `ZVAL` as a property. However, when carrying out partial charge calculations (e.g. a Bader analysis), having information about the `ZVAL` is crucial. Since the `ZVAL` isn't stored anywhere other than the POTCAR file or OUTCAR file itself, this means that downstream codes like Atomate/Atomate2 don't have the `ZVAL` stored anywhere in the tabulated data and it can be annoying to access. 

In short, we should add the `ZVAL` as a property to the `Potcar()` object. Up for debate, but perhaps it should also be added to the `spec` property.

**Comment by arosen93** [comment]
Sorry, I see this is under `nelectrons`. It just doesn't make its way into the POTCAR spec.

**Comment by shyuep** [comment]
I have no objection to adding a ZVAL as a alternative name or in the spec.

**Comment by arosen93** [comment]
Thanks! I might consider adding it to the spec. But I also think it's mostly a non-issue if the data is present elsewhere already.
